apps/spark/src/main/java/com/linkedin/openhouse/jobs/client/TablesClient.java

```diff
@@ -268,6 +268,7 @@ protected TableMetadata mapTableResponseToTableMetadata(GetTableResponseBody res
         .creator(creator)
         .dbName(responseBody.getDatabaseId())
         .tableName(responseBody.getTableId())
+        .creationTime(responseBody.getCreationTime())
```

Comment:
when did we add creation time? do some tables return null for this field? @autumnust

Reply:
Considering possibility of default (0) values for creationTime as a result of searchApi, I have changed the implementation to depend on getTable API which loads the table and gets correct table creationTime. 
Added null check of safety.

Reply:
creationTime and lastModTime was added long time ago and in the early draft of table api spec https://docs.google.com/document/d/1E_1Yr_ZQc2iZraL_--9lcREBt-YADyj-4zdgCyAxNtY/edit#bookmark=id.jp3ymapwnytk

Reply:
`creationTime` is present in GetTableResponseBody, but `tables/search` API does not return this filed populated and has default values (0). All fields except tableId and databaseId are defaulted.
Ideally we have reused `GetTableResponseBody` in search response which could be error prone. Ideally should create a new `searchResponseBody`.